tests/test_mac_address.py

```diff
@@ -66,7 +69,7 @@ def test_file_exists_exhaustion(tmpdir):
         with pytest.raises(mac_address.MacAddressException):
             mac_address.reserve_unique_mac_address(str(tmpdir))
     finally:
-        [p.kill() for p in flock_processes]
```

Comment:
technically unrelated to this change, but I noticed that this test was leaving around tons of orphaned processes